docs/tutorials/08_Initiators_&_Deleters.py

```diff
@@ -0,0 +1,241 @@
+            measurement_set.add(TrueDetection(state_vector=measurement,
+                                              groundtruth_path=truth,
+                                              timestamp=truth_state.timestamp))
+
+            # Generate clutter at this time-step
+            truth_x = truth_state.state_vector[0]
+            truth_y = truth_state.state_vector[2]
+            for _ in range(np.random.randint(2)):
+                x = uniform.rvs(truth_x - 10, 20)
+                y = uniform.rvs(truth_y - 10, 20)
+                measurement_set.add(Clutter(np.array([[x], [y]]), timestamp=timestamp))
+    all_measurements.append(measurement_set)
+
+# Plot detections and clutter
+for set_ in all_measurements:
+    # Plot actual detections.
+    ax.scatter([state.state_vector[0] for state in set_ if isinstance(state, TrueDetection)],
+               [state.state_vector[1] for state in set_ if isinstance(state, TrueDetection)],
+               color='g')
+    # Plot clutter.
+    ax.scatter([state.state_vector[0] for state in set_ if isinstance(state, Clutter)],
+               [state.state_vector[1] for state in set_ if isinstance(state, Clutter)],
+               color='y',
+               marker='2')
+fig
+
+# %%
+# Creating a Tracker
+# ------------------
+# We'll now create the tracker components as we did with the multi-target examples previously.
+
+from stonesoup.predictor.kalman import KalmanPredictor
+predictor = KalmanPredictor(transition_model)
+
+from stonesoup.updater.kalman import KalmanUpdater
+updater = KalmanUpdater(measurement_model)
+
+from stonesoup.hypothesiser.distance import DistanceHypothesiser
+from stonesoup.measures import Mahalanobis
+hypothesiser = DistanceHypothesiser(predictor, updater, measure=Mahalanobis(), missed_distance=3)
+
+from stonesoup.dataassociator.neighbour import GNNWith2DAssignment
+data_associator = GNNWith2DAssignment(hypothesiser)
+
+# %%
+# Creating a Deleter
+# ------------------
+# Here we are going to create an error based deleter, which will delete any :class:`~.Track` where
+# trace of the covariance is over a certain threshold i.e. when we have a high uncertainty. This
+# simply requires a threshold to be defined, that will depend on units and number of dimensions of
+# your state vector.
+from stonesoup.deleter.error import CovarianceBasedDeleter
+deleter = CovarianceBasedDeleter(4)
+
+# %%
+# Creating an Initiator
+# ---------------------
+# Here we are going to use a measurement based initiator, which will create a track from the
+# unassociated :class:`~.Detection` objects. This still requires a prior to be defined for the
+# :class:`~.Track`, but elements of the state vector that are measured are replaced by that of the
+# measurement, and the measurement's uncertainty (these defined by the
+# :class:`~.MeasurementModel`). In this example, as our sensor measures position, we only need to
+# be concerned about the values provided for the velocity and it's variance.
```

Comment:
This took a few reads to get my head around it. Could it be rewritten? 
```suggestion
# unassociated :class:`~.Detection` objects. A prior is still required to initialise the elements 
# of the state vector that cannot be initialised from the measurements and the measurement's 
# uncertainty (these defined by the :class:`~.MeasurementModel`). In this example, as our 
# sensor measures position, so the prior is needed to for the velocity and it's variance.
```
I don't think mine is any clearer 👎

Reply:
I've made an attempt to rewrite.